kahypar/partition/initial_partition.h

```diff
@@ -63,12 +64,9 @@ inline Context createContext(const Hypergraph& hg,
   context.local_search = context.initial_partitioning.local_search;
 
   // Hypergraph depending parameters
-  context.coarsening.contraction_limit = context.coarsening.contraction_limit_multiplier
-                                         * context.initial_partitioning.k;
-  context.coarsening.hypernode_weight_fraction = context.coarsening.max_allowed_weight_multiplier
-                                                 / context.coarsening.contraction_limit;
-  context.coarsening.max_allowed_node_weight = ceil(context.coarsening.hypernode_weight_fraction
-                                                    * hg.totalWeight());
+  context.coarsening.contraction_limit = std::numeric_limits<HypernodeID>::max();
```

Comment:
Hm... Unfortunately,  this is not correct.  These values actually need to be set up correctly for initial partitioning to work as expected.